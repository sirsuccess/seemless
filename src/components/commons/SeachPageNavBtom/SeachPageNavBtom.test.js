import React from "react";
import renderer from "react-test-renderer";

import Component from "./SeachPageNavBtom";

it("renders correctly when called ", () => {
  const tree = renderer.create(<Component />).toJSON();
  expect(tree).toMatchSnapshot();
});
