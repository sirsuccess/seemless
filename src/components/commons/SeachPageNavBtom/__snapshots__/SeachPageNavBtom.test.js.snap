// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
<div
  className="SeachPageNavBtom"
>
  <ul>
    <li>
      <i
        class="fa fa-search"
      />
       All
    </li>
    <li>
      <i
        class="fa fa-image"
      />
       Images
    </li>
    <li>
      <i
        class="fa fa-map-marker-alt"
      />
      Maps
    </li>
    <li>
      <i
        class="fa fa-play-circle"
      />
       Videos
    </li>
    <li>
      <i
        class="fa fa-newspaper"
      />
       News
    </li>
    <li>
      <i
        class="fa fa-ellipsis-v"
      />
       More
    </li>
  </ul>
  <div
    className="tool"
  >
    <span>
      Settings
    </span>
    <span>
      Tools
    </span>
  </div>
</div>
`;
